jenkins/cut-release

```diff
@@ -103,5 +110,5 @@ update_and_or_cut_branch_for_repository() {
 
 # A copy will be preserved as a build artifact.
 rm -f "${output_file}"
-touch "${output_file}"
+touch "${output_file}" || true
```

Comment:
This can fail the first time you run the script without this,
since the parent directory doesn't exist yet.